infra/config/validation.yaml

```
@@ -3,70 +3,82 @@
-        # For changes that optimize speed or resource usage.
-      - Performance
-        # For changes that improve the structure or separation of concerns.
-      - Modularity
-        # When refactoring to make the code easier to understand.
-      - Readability
-        # For improvements that simplify future updates or maintenance.
-      - Maintainability
+      - Performance     # For changes that optimize speed or resource usage.
+      - Modularity      # For changes that improve structure/separation of concerns.
+      - Readability     # When refactoring to improve clarity.
+      - Maintainability # For improvements that simplify future maintenance.
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)